ui/.env.production

```diff
@@ -5,3 +5,4 @@ REACT_APP_SENTRY_DSN=https://10930d3b0a8d427cad2147d6d845be56@o226447.ingest.sen
 REACT_APP_SENTRY_ENABLED=true
 # REACT_APP_RELEASE_VERSION is set in config-overrides.js
 REACT_APP_RELEASE_VERSION=unknown
+REACT_APP_MIXPANEL_TOKEN=
```

Comment:
@baurine Could you provide the token for this?

Reply:
It looks like we need to provide the token from PD's environment to prevent the token leaks. There are some unpredictable telemetry in the previous mixpanel project.

Reply:
The previous prod token was leaked https://github.com/pingcap/tidb-dashboard/pull/895/files#diff-f7e6bf7e30ccb470c218b3ce574c31e07e802d50c78d4a9802fa7c7ce2fcd14eL3

I found PD was embedded the dashboard static assets from tidb-dashboard's release, it seems we can add env variables to tidb-dashboard repo's workflow instead of pd.

Reply:
We need to make sure that the telemetry works.